docs/helpers.rst

```diff
@@ -69,7 +77,11 @@ Markers
  Test classes that subclass :class:`django.test.TestCase` will have access to
  the database always to make them compatible with existing Django tests.
  Test classes that subclass Python's :class:`unittest.TestCase` need to have
- the marker applied in order to access the database.
+ marker applied in order to access the database.
+
+.. _rollback emulation: https://docs.djangoproject.com/en/stable/topics/testing/overview/#rollback-emulation
+.. _django.test.TestCase: https://docs.djangoproject.com/en/dev/topics/testing/overview/#testcase
```

Comment:
intersphinx references should be used when possible (see my example suggestion above,

```
:class:`django.test.TransactionTestCase`
```